src/cityreader/cityreader.py

```diff
@@ -1,6 +1,18 @@
 # Create a class to hold a city location. Call the class "City". It should have
 # fields for name, lat and lon (representing latitude and longitude).
 
+import csv
+
+class City():
+  def __init__(self, name, lat, lon):
+    self.name = name
+    self.lat = lat
+    self.lon = lon
+  
+  def __str__(self):
```

Comment:
Nice use of this method to debug. Why did you use this method instead of `__repr__`?